clustering/web/undertow/src/main/java/org/wildfly/clustering/web/undertow/logging/UndertowClusteringLogger.java

```diff
@@ -58,4 +58,8 @@
     @Message(id = 8, value = "No distributable single sign-on management provider found for %s; using legacy provider based on static configuration")
     @LogMessage(level = Level.WARN)
     void legacySingleSignOnProviderInUse(String name);
+
+    @Message(id = 9, value = "Session %s is being modified when the response is closed (e.g. after HttpServletResponse sendRedirect or sendError). " +
```

Comment:
This should probably read: "Session %s invalidation attempted after the response was committed (e.g. after HttpServletResponse.sendRedirect(...) or sendError(...)."

Reply:
Done!